packages/neutrino-middleware-hot/index.js

```diff
@@ -1,3 +1,5 @@
-const { HotModuleReplacementPlugin } = require('webpack');
+const { HotModuleReplacementPlugin, NamedModulesPlugin } = require('webpack');
 
-module.exports = ({ config }) => config.plugin('hot').use(HotModuleReplacementPlugin);
+module.exports = ({ config }) => config.plugin('hot')
+  .use(HotModuleReplacementPlugin)
+  .use(NamedModulesPlugin);
```

Comment:
Using `use` twice on the same plugin will replace the first with the second. You want to create another plugin instance and use that:

```js
.plugin('hot')
  .use(HotModuleReplacementPlugin)
  .end()
.plugin('named-modules')
  .use(NamedModulesPlugin);